src/components/fields/e-signature-field/e-signature-field.tsx

```diff
@@ -103,25 +110,44 @@ export const ESignatureField = (props: IGenericFieldProps<IESignatureFieldSchema
 			},
 		});
 		setLoadingProgress(null);
-
 		return response;
 	};
 
 	// =============================================================================
 	// RENDER FUNCTIONS
 	// =============================================================================
+	const renderError = () => {
+		if (!error?.message) return null;
+
+		return (
+			<ErrorWrapper weight="semibold">
+				{error.message}
+				{uploadErrorCount > 0 && (
+					<TryAgain type="button" onClick={() => handleChange(signatureDataURL)}>
```

Comment:
whether try again should still be clickable when retry is in progress

Reply:
updated to hide error messages while uploading. this applies to other validation error messages too.